pkg/filters/context.go

```diff
@@ -167,7 +169,8 @@ func (filter *eventCtxFilter) Filter(evt trace.Event) bool {
 		filter.podNSFilter.Filter(evt.PodNamespace) &&
 		filter.podUIDFilter.Filter(evt.PodUID) &&
 		filter.tidFilter.Filter(int64(evt.ThreadID)) &&
-		filter.uidFilter.Filter(int64(evt.UserID))
+		filter.uidFilter.Filter(int64(evt.UserID)) &&
+		filter.syscallFilter.Filter(evt.Syscall)
```

Comment:
This should probably be higher in the chain so it will get short circuited faster.